target/linux/ar71xx/image/tp-link.mk

```diff
@@ -426,7 +426,7 @@ TARGET_DEVICES += tl-wpa8630
 define Device/tl-wr703n-v1
     $(Device/tplink-4mlzma)
     DEVICE_TITLE := TP-LINK TL-WR703N
-    DEVICE_PACKAGES := kmod-usb-core kmod-usb2
+    DEVICE_PACKAGES := kmod-usb-core kmod-usb-gadget kmod-usb-chipidea
     BOARDNAME := TL-WR703N
```

Comment:
users will most likely expect usb support on the port and not gadget. can you drop this part from the patch please.